Dockerfile

```diff
@@ -20,17 +19,17 @@ ENV        NODE_PATH=/calypso/server:/calypso/client
 COPY       ./env-config.sh /tmp/env-config.sh
 RUN        bash /tmp/env-config.sh
 
+# Update npm
+RUN npm install --global npm@6.1
```

Comment:
I'd left this as `npm@6.1` in #25372, we probably want to pin exact version `6.1.0` here?